src/corespotlight.cs

```diff
@@ -1075,5 +1092,14 @@ interface CSSearchQuery {
 		[Export ("cancel")]
 		void Cancel ();
 	}
+
+	[NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject))]
+	interface CSImportExtension : NSExtensionRequestHandling
```

Comment:
doc suggest this is to be subclassed inside an extension
IOW it's not usable as-is inside apps
might be worth making it `abstract`

Reply:
We want the abstract keyword because CSImportExtension will be subclassed and we want the user to implement it's properties and methods? (just Update in this case)

Reply:
Adding `[Abstract]` has its own set of problems (https://github.com/xamarin/xamarin-macios/issues/4969), although those might not necessarily apply to this case.